src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,14 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private String model;
```

Comment:
The field 'model' should have an access modifier to encapsulate the field properly. Consider using 'private' to restrict direct access.